packages/framework-core/src/booster-token-verifier.ts

```diff
@@ -63,13 +63,16 @@ export class BoosterTokenVerifier {
   }
 
   private tokenToUserEnvelope(decodedToken: any): UserEnvelope {
-    const username = decodedToken.email ?? decodedToken.phone_number
+    const username = decodedToken?.email || decodedToken?.phone_number || decodedToken.sub
```

Comment:
If for some reason the provider doesn't gives the email or phone_number will use the sub as the username.